linkerd/cache/src/lib.rs

```diff
@@ -34,7 +34,7 @@ type Services<T, S> = HashMap<T, (S, Weak<()>)>;
 
 impl<T, N> Cache<T, N>
 where
-    T: Eq + Hash,
+    T: Eq + Hash + Send + 'static,
```

Comment:
Does the cache actually require this? Or from debugging?

Reply:
The `Service` impl for `Cache` already has this bound on `master-tokio-0.2`:
https://github.com/linkerd/linkerd2-proxy/blob/1755e2d381a2a341a233a9fe3c13cfbcc78b0170/linkerd/cache/src/lib.rs#L66

It's required because `Lock::poll_acquire` requires that the type inside the lock be `'static` in order to box the `tokio::sync::Mutex::lock` future.